### DISCUSSION TITLE
No task template appear in new view I created

### DISCUSSION THREAD

**Discussion by AishahFatihahh** [semaphoreui/semaphore#1462] [Q&A]
**Describe the issue**
I want to create new task in new view I created (PROD, QAT, UAT). After create the template in the view, it did not appear in that specific view. It only appear in 'ALL' view.

**Screenshots**

![image](https://github.com/ansible-semaphore/semaphore/assets/138760516/bc264105-ffef-4fdf-8945-a374c8300e8e)

![image](https://github.com/ansible-semaphore/semaphore/assets/138760516/8e11e290-2fdf-436b-90d8-f15ba7d82592)

**Expected behaviour**
Task template created in Prod should appear in Prod

**Comment by ccrembil** [comment]
Did you complete the "View" field of the Task Template? (options PROD, QAT and UAT should appear there)
